change_log.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+docker run -it --rm -e CHANGELOG_GITHUB_TOKEN=$CHANGELOG_GITHUB_TOKEN -v "$(pwd)":/usr/local/src/your-app githubchangeloggenerator/github-changelog-generator \
+    --user apache \
+    --project arrow-rs \
+    --since-commit 2021-04-20 \
+    --future-release 4.0.1
+sed -i "s/\\\n/\n\n/" CHANGELOG.md
```

Comment:
I wonder if this should be something like

```suggestion
sed -i -e "s/\\\n/\n\n/" CHANGELOG.md
```

Though I am not a `sed` 🧙

Reply:
This was due to a bug on the generator, https://github.com/github-changelog-generator/github-changelog-generator/issues/962, that it was fixed and released in less than 24h (big kudos to @olleolleolle for this 💯 ). I need to check that the new docker image works as intended, so that we can remove this hack. 👍